resources/views/admin/pages/products/show.blade.php

```diff
@@ -0,0 +1,78 @@
+@extends('admin.layout.master')
+@section('title',__('product.admin.show.title') )
+@section('body')
+<!-- Hover Rows -->
+<div class="row clearfix">
+  @include('admin.includes.message')
+  <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+    <div class="card">
+      <div class="body">
+        <div class="header">
+          <h2>{{__('product.admin.show.details_product')}}</h2>
+          <a href="{{ route('admin.products.create') }}"
+            class="btn bg-green waves-effect" style="margin-top: 30px;"> <i
+            class="material-icons">playlist_add</i> <span>{{ __('product.admin.show.create_product') }}</span>
+          </a>
+        </div>
+        <div class="row clearfix">
+          <div class="col-sm-3">
+            <h4>{{__('product.admin.show.name')}}:</h4>
+          </div>
+          <div class="col-sm-9">
+            <p class="font-22">{{ $product->name }}</p>
+          </div>
+        </div>
+        <div class="row clearfix">
+          <div class="col-sm-3">
+            <h4>{{__('product.admin.show.describe')}}:</h4>
+          </div>
+          <div class="col-sm-9">
+            <p class="font-20">{{ $product->describe }}</p>
+          </div>
+        </div>
+        <div class="row clearfix">
+          <div class="col-sm-3">
+            <h4>{{__('product.admin.show.price')}}:</h4>
+          </div>
+          <div class="col-sm-9">
+            <p class="font-20">{{ $product->price }} VND</p>
+          </div>
+        </div>
+        <div class="row clearfix">
+          <div class="col-sm-3">
+            <h4>{{__('product.admin.show.category')}}:</h4>
+          </div>
+          <div class="col-sm-9">
+            <p class="font-20">{{ $product->category_id }}</p>
+          </div>
+        </div>
+        <div class="row clearfix">
+          <div class="col-sm-3">
+            <h4>{{__('product.admin.show.store')}}:</h4>
+          </div>
+          <div class="col-sm-9">
+            <p class="font-20">{{ $product->store_id }}</p>
+          </div>
+        </div>
+        <div class="row clearfix">
+          <div class="col-sm-3">
+            <h4>{{__('product.admin.show.image')}}:</h4>
+          </div>
+          <div class="col-sm-9">
+            <img class="img-responsive thumbnail" src="images/{{ $product->images->first()->path}}">
```

Comment:
define một cái  imagePathAttribute() trong model để dùng
tham khảo 
https://laravel.com/docs/5.5/eloquent-mutators#accessors-and-mutators